docs/endpoints/constructorStandings.md

```diff
@@ -0,0 +1,205 @@
+[← Documentation Home](/docs/README.md)
+# Constructor Standings
+
+Gets a season's constructors standings 
+
+**URL** : `/ergast/f1/{YEAR}/constructorstandings/`
+
+[Available Query Parameters](./README.md#query-parameters)
+
+---
+
+## Route Parameters:
+
+### Season \*\*REQUIRED\*\*
+
+**Filters for the constructors standing of a specified season. Year numbers are valid as is `current` to get the current seasons constructors standings**
+
+`/{season}/` -> ex: `/ergast/f1/2024/constructorstandings/`
+
+**Note**: To utilize the `season` parameter it needs to be the first route after `/ergast/f1/`
+
+---
+
+### Round
+
+**Filters for the constructors standings after a specified round in a specific season. Round numbers 1 -> n races are valid as well as `last`**
+
+`/{season}/{roundNumber}/` -> ex: `/ergast/f1/2024/5/constructorstandings/`
```

Comment:
I'm open to suggestions on what it should be, but we should name things consistently, e.g. `round` vs `roundNumber`, `season` vs `YEAR`.

For inspiration, this is what the value are called internally. We don't necessarily have to use them
https://github.com/jolpica/jolpica-f1/blob/8424ed415102dfa24de5b8a85faf605e9a5ee995/jolpica_api/ergastapi/urls.py#L44-L55

Not sure, for example, whether in driver filters we should use `driver`, `driver_ref` or `driver_reference`,

Reply:
Honestly last night I was slightly struggling with a few things like "what did I call it there" happy to refer to it as something more consistent.

Reply:
Do what you think is best, I'll find some time to do a full read through of everything this evening.

Reply:
Hey @jolpica I found my little notepad I was scribbling thoughts onto. 

The reason I was using driverId for the field after drivers/ is because if you use a drivers filter. What is shown in the response for: https://api.jolpi.ca/ergast/f1/drivers/hamilton/races/ is this:
<img width="229" alt="image" src="https://github.com/user-attachments/assets/285bc8ae-f62a-46e1-9ae6-694bf1f35d66">

So I was using driverId, constructorId, season, etc because this is the name of the property that is shown in the response.

As for fastest, grid, results and status I slightly modified them because the fields are technically just called "fastest", "grid", "position" and "status" respectively and that doesn't properly capture what the field is meant oto mean.

So I changed:
|route|from|to|
|----|-----|-----|
|/fastest   |  `/{fastest}`    |    `/{lapRank}` |
|/grid     |     `/{grid}`         |   `/{gridPosition}` |
|/results    |  `/{position}`  |   `/{finishPosition}`|
|/status    |   `/{status}`     |   `/{statusId}`|

as I felt they are a bit more clear as to what they do/are at a simple glance. (like if you saw `status` you might think "Disqualified", or "Accident" would be valid, but `statusId` should tell you that you need to use a specific ID for a status.

If someone were to look for the status in the `/status/` endpoint they will see the status objects where there is a field called `statusId`and `status` and the `status`field here is actually not valid. 

You cant do: https://api.jolpi.ca/ergast/f1/status/Disqualified/races

but you can: https://api.jolpi.ca/ergast/f1/status/2/races

```
"StatusTable": {
      "Status": [
        {
          "statusId": "1",
          "count": "7592",
          "status": "Finished"
        },
        {
          "statusId": "2",
          "count": "146",
          "status": "Disqualified"
        },
        {
          "statusId": "3",
          "count": "1059",
          "status": "Accident"
        },
```